### ISSUE TITLE
do not force the bg-gray background for sections

### ISSUE THREAD

**Issue by aoloe** [getgrav/grav-theme-quark#93]
currently, there is no way to disable the `bg-gray` class that is added to the modular sections

on top of it, the gray background color is marked as important.

on the one side there should be a way to disable the class (or, even better, the class should be opt in).
(i could not find any)

on the other side, i don't see what is so !important with that background color...

**Comment by mrichar1** [comment]
Just to add this seems to only be applied if `header-dark` class is set - if you disable this in the UI/`quark.yaml`, and ensure that it also isn't enforced using `body_classes` in the page frontmatter then these values can be overridden:

https://github.com/getgrav/grav-theme-quark/blob/develop/scss/theme/_header.scss#L21-L36

**Comment by aoloe** [comment]
@mrichar1 , i don't think you can add css class to html / twig templates from a scss file...
but i'm really no scss expert. just using common sense...

**Comment by mrichar1** [comment]
Sorry to be unclear - I wasn't meaning that you need to change/override  `_header.scss` - I was highlighting that the `!important` settings are applied if `.header-dark` is set, and that this can be enabled either in the theme's `quark.yaml`/UI OR by adding it to the `bg_classes` setting in the page frontmatter.

**Comment by aoloe** [comment]
the most important part to is that there is a class called `bg-gray` applied to an "unrelated" item.

if feels wrong to redefine `bg-gray` as being white.

and i also think that a css file that is supposed to be extended by the final user should not be marked as `!important`.

i will have to check if disabling the "header-dark" set has an advantage to me, but the project  that "raised" issue is now published and i'm not currently working on it right now.

**Comment by NicoHood** [comment]
I also stumbled across this. The `bg-grey` class should be used and "normal" css styling should be applied to those elements in my opinion.
